### PR TITLE
[JENKINS-37613] Unify installations naming criteria. mavenInstallation -> maven

### PR DESCRIPTION
**Pull request by alvarolobato**
@reviewbybees

### CODE REVIEW

**Comment by cyrille-leclerc** [comment]
👍

**Comment by jtnord** [comment]
:-1: What no JIRA - (as this will break backwards compatability...).

Personally I think `jdkInstallation` is too wordy - I would have renamed `mavenInstallation` to `maven`

will remove the :-1: when there is a JIRA (yes I kno wthis is a BETA - but this would break anyone that was using this...)

**Comment by cyrille-leclerc** [comment]
@alvarolobato @jtnord I am fine with both "standardizations" (suffix by "installation" or omit "installation"). I am fine with creating a Jira, James is right, this is a backward incompatibility (whatever is the standardization).

**Comment by alvarolobato** [comment]
Jira attached and changed to a less verbose form without 'installation'

**Comment by cyrille-leclerc** [comment]
👍

**Comment by alvarolobato** [comment]
@jtnord can you re-evaluate the bug 😉

**Comment by jtnord** [comment]
:bee:
